packages/styles/components/_c.pagination.scss

```diff
@@ -0,0 +1,58 @@
+.mc-pagination {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__button {
+    @include unstyle-button();
+    @include set-button-theme(get-token(color, button, 'solid-neutral'));
+    @include set-button-layout();
+
+    box-sizing: border-box;
+    min-height: $mu300;
+    min-width: $mu300;
+    margin: 0 $mu075;
```

Comment:
~~margin: 0 $mu075;~~
I think you can take it off the buttons and put it on the select container instead.